ros_buildfarm/templates/status/js/setup.js

```diff
@@ -271,14 +271,16 @@ function filter_table() {
               break;
             }
           }
+          // replace OR to RegExp syntax(|)
+          queries[i] = queries[i].replace(/\s*\OR\s*/, '|');
           if (is_known_query) {
             // search in full row html
-            if (row[0].indexOf(queries[i]) == -1) return null;
+            if (! RegExp(queries[i]).test(row[0])) return null;
```

Comment:
Switching the query string into a regex might affect strings which have special meaning within a regex. I am not sure if that is desired in all cases? If so it should probably be mentioned in the help / hover text of the search field.